core/scripts/Voting.js

```diff
@@ -13,29 +13,41 @@ const argv = require("minimist")(process.argv.slice(), { string: ["network"] });
 
 const SUPPORTED_IDENTIFIERS = {
   BTCUSD: {
-    dataSource: "CryptoCompare",
-    identifiers: { first: "BTC", second: "USD" }
+    numerator: {
+      dataSource: "CryptoCompare",
+      identifiers: { first: "BTC", second: "USD" }
+    }
   },
   "Telegram SAFT": {
-    dataSource: "Constant",
-    value: "100"
+    numerator: {
+      dataSource: "Constant",
+      value: "100"
+    }
   },
   "Custom Index (1)": {
-    dataSource: "Constant",
-    value: "1"
+    numerator: {
+      dataSource: "Constant",
+      value: "1"
+    }
   },
   "Custom Index (100)": {
-    dataSource: "Constant",
-    value: "100"
+    numerator: {
+      dataSource: "Constant",
+      value: "100"
+    }
   },
   TSLA: {
-    dataSource: "IntrinioEquities",
-    // TODO: TSLA not supported in sandbox, but presumably is supported in their prod instance.
-    symbol: "AAPL"
+    numerator: {
+      dataSource: "IntrinioEquities",
+      // TODO: TSLA not supported in sandbox, but presumably is supported in their prod instance.
```

Comment:
Can we switch this out for the correct symbol?

Reply:
Done.